tests/e2e/registry/test_registry_pull_and_push_images_to_registry.py

```diff
@@ -1,37 +1,37 @@
 import pytest
 import logging
 
-from ocs_ci.ocs.constants import OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
-from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs import ocp, constants
 from ocs_ci.ocs.registry import (
-    validate_registry_pod_status,
-    image_pull, image_push, image_list_all, image_rm,
-    check_image_exists_in_registry
+    validate_registry_pod_status, image_pull_and_push
 )
-from ocs_ci.framework.testlib import E2ETest, workloads, tier1
+from ocs_ci.framework.testlib import E2ETest, workloads
 
 log = logging.getLogger(__name__)
-IMAGE_URL = 'docker.io/library/busybox'
 
 
 @workloads
-@tier1
 class TestRegistryPullAndPushImagestoRegistry(E2ETest):
     """
     Test to pull and push images
     to registry backed by OCS
     """
 
     @pytest.fixture(autouse=True)
-    def teardown(self, request):
+    def setup(self, request):
         """
         Clean up svt
```

Comment:
The function now was changed to setup, please update the doc string

Reply:
done